graphs/breathfirstsearch/breathfirstsearch.go

```diff
@@ -1,73 +1,31 @@
 package breathfirstsearch
 
-// GetIdx ...
-func GetIdx(target int, nodes []int) int {
-	for i := 0; i < len(nodes); i++ {
-		if nodes[i] == target {
-			return i
-		}
-	}
-	return -1
-}
+/*
 
-// NotExist ...
-func NotExist(target int, slice []int) bool {
-	for i := 0; i < len(slice); i++ {
-		if slice[i] == target {
-			return false
-		}
-	}
-	return true
-}
+	Breadth First Search
```

Comment:
Always start the comment with the name of the function, in this case

```suggestion
	BreadthFirstSearch <description goes here>
```